src/masternode-budget.cpp

```diff
@@ -1001,7 +1001,7 @@ void CBudgetManager::ProcessMessage(CNode* pfrom, std::string& strCommand, CData
         if (Params().NetworkID() == CBaseChainParams::MAIN) {
             if (nProp == 0) {
                 if (pfrom->HasFulfilledRequest("mnvs")) {
-                    LogPrintf("mnvs - peer already asked me for the list\n");
+                    LogPrint("masternode","mnvs - peer already asked me for the list\n");
```

Comment:
post-merge: This should have been left as `LogPrintf()` as it results in a `Misbehaving` message.